docs/howtos/state_params.rst

```diff
@@ -1,6 +1,6 @@
 .. image:: https://colab.research.google.com/assets/colab-badge.svg
-   :target: https://github.com/google/flax/blob/main/docs/howtos/state_params_howto.ipynb
-   
+   :target: https://colab.research.google.com/github/google/flax/blob/main/docs/notebooks/state_params_howto.ipynb
```

Comment:
nit: for consistency consider removing the file to `notebooks/state_params.ipynb`

Reply:
I suppose you mean renaming? Ok I'll do that.